app/controllers/staff/organisations_controller.rb

```diff
@@ -12,16 +12,12 @@ def show
 
     @organisation_presenter = OrganisationPresenter.new(organisation)
 
-    fund_activities = FindFundActivities.new(organisation: organisation, user: current_user).call
-    programme_activities = FindProgrammeActivities.new(organisation: organisation, user: current_user).call
-    project_activities = FindProjectActivities.new(organisation: organisation, user: current_user).call
-    third_party_project_activities = FindThirdPartyProjectActivities.new(organisation: organisation, user: current_user).call
+    project_activities = FindProjectActivities.new(organisation: organisation, user: current_user).call(eager_load_parent: false)
+    third_party_project_activities = FindThirdPartyProjectActivities.new(organisation: organisation, user: current_user).call(eager_load_parent: false)
     submissions = SubmissionPolicy::Scope.new(current_user, Submission).resolve.includes(:organisation, :fund)
 
     respond_to do |format|
       format.html do
-        @fund_activities = fund_activities.map { |activity| ActivityPresenter.new(activity) }
```

Comment:
Is there a case where we show any level of activity on the home page? I wonder if all of the finding of these activities is only required when the user requests XML?

Reply:
We use these to decide whether we show the download xml button, we agreed finding a good place for these buttons is low priority right now, we can definately make further optimisations on these page as it morphs into the home page again or becomes a true `Organisation` show view.